<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QDialog" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(26, 25, 40);</string>
  </property>
  <widget class="QLabel" name="greet">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>1301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/aaa.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>130</y>
     <width>1331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>20</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>700</y>
     <width>1351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/ppy.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>601</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 0, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="viewFriendsList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Unispace&quot;;
background-color: rgb(85, 0, 255);
</string>
    </property>
    <property name="text">
     <string>Friends
List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="memories">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Unispace&quot;;
background-color: rgb(85, 0, 255);
</string>
    </property>
    <property name="text">
     <string>Memories</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createPost">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>291</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Unispace&quot;;
background-color: rgb(85, 0, 255);
</string>
    </property>
    <property name="text">
     <string>Create Post</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switchUser">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>0</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Unispace&quot;;
background-color: rgb(85, 0, 255);
</string>
    </property>
    <property name="text">
     <string>Switch
User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewLikedPages">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>140</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Unispace&quot;;
background-color: rgb(85, 0, 255);
</string>
    </property>
    <property name="text">
     <string>Liked
Pages</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewTimeline">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>290</y>
      <width>291</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Unispace&quot;;
background-color: rgb(85, 0, 255);
</string>
    </property>
    <property name="text">
     <string>Timeline</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewHome">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 24pt &quot;Unispace&quot;;
background-color: rgb(85, 0, 255);
</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>greet</zorder>
  <zorder>label</zorder>
  <zorder>frame_3</zorder>
  <zorder>frame_4</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
